app/models/ext_management_system.rb

```diff
@@ -513,10 +513,7 @@ def perf_rollup_parents(interval_name = nil)
   end
 
   def perf_capture_enabled?
-    return @perf_capture_enabled unless @perf_capture_enabled.nil?
-    return @perf_capture_enabled = true if ems_clusters.any?(&:perf_capture_enabled?)
-    return @perf_capture_enabled = true if hosts.any?(&:perf_capture_enabled?)
-    @perf_capture_enabled = false
+    @perf_capture_enabled ||= ems_clusters.any?(&:perf_capture_enabled?) || host.any?(&:perf_capture_enabled?)
```

Comment:
`||=` can't be used for the nil check because the value is a boolean (i.e. false will cause it to evaluate the RHS again)